am-core-web-service/src/main/resources/db/migration/V3_careerAddCode.sql

```diff
@@ -0,0 +1,9 @@
+ALTER TABLE career
+    ADD COLUMN code varchar(10);
```

Comment:
deberia ser algo como alter table career add column code varcha(10) not null; pero se puede aniadir valores por defecto para los datos que ya tiene. Averigualo eso y borra lo de abajo. Con una sola consulta es suficiente